GCP Marketplace workflow fails

### Description

Workflow fails due to the inclusion of graphql section in the Application resource.

https://github.com/hashgraph/hedera-mirror-node/actions/runs/4327737241/jobs/7556942392

```
2023-03-03 22:45:18 (149 MB/s) - 'yq_linux_amd64' saved [10180037/10180037]

Removing intermediate container e0f88f6a40d9
 ---> 4e119412ec1f
Step 3/14 : WORKDIR /app
 ---> Running in c9d7cceb498a
Removing intermediate container c9d7cceb498a
 ---> 056c3dfaf274
Step 4/14 : COPY . ./
 ---> fd0cb093e2a8
Step 5/14 : RUN yq -i '. *= load("marketplace/gcp/values.yaml")' hedera-mirror/values.yaml
 ---> Running in cf9d1cc7f571
Removing intermediate container cf9d1cc7f571
 ---> 97814e5a2dee
Step 6/14 : ARG TAG
 ---> Running in 2e0c8fdd32de
Removing intermediate container 2e0c8fdd32de
 ---> 4e10184bf910
Step 7/14 : RUN sed -i "s/\$TAG/$TAG/g" marketplace/gcp/schema.yaml
 ---> Running in 7badde9f250a
Removing intermediate container 7badde9f250a
 ---> a06668a71ff8
Step 8/14 : RUN helm dependency update hedera-mirror
 ---> Running in 560e0ef74aaf
Getting updates for unmanaged Helm repositories...
...Successfully got an update from the "https://charts.bitnami.com/bitnami" chart repository
...Successfully got an update from the "https://charts.bitnami.com/bitnami" chart repository
...Successfully got an update from the "https://charts.bitnami.com/bitnami" chart repository
Saving 10 charts
Downloading postgresql from repo https://charts.bitnami.com/bitnami
Downloading postgresql-ha from repo https://charts.bitnami.com/bitnami
Downloading redis from repo https://charts.bitnami.com/bitnami
Deleting outdated charts
Removing intermediate container 560e0ef74aaf
 ---> 61[327](https://github.com/hashgraph/hedera-mirror-node/actions/runs/4327737241/jobs/7556942392#step:12:328)ae2119e
Step 9/14 : RUN helm template hedera-mirror -f hedera-mirror/values.yaml
 ---> Running in a3b2d42ef9e6
Error: YAML parse error on hedera-mirror/templates/application.yaml: error converting YAML to JSON: yaml: line 63: did not find expected comment or line break

Use --debug flag to render out invalid YAML
The command '/bin/sh -c helm template hedera-mirror -f hedera-mirror/values.yaml' returned a non-zero code: 1

Error: Process completed with exit code 1.
```

### Steps to reproduce

Run the GCP Marketplace workflow that runs during production release

### Additional context

_No response_

### Hedera network

other

### Version

0.76.0

### Operating system

None

--- a/charts/hedera-mirror/templates/application.yaml
+++ b/charts/hedera-mirror/templates/application.yaml
@@ -55,16 +55,6 @@ spec:
       - name: Hedera Mirror Node Team
         email: mirrornode@hedera.com
     notes: |
-      {{- if .Values.graphql.enabled -}}
-      To access the GraphQL API:
-
-        GRAPHQL_IP=$(kubectl get service/{{ .Release.Name }}-graphql -n {{ include "hedera-mirror.namespace" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-
-        curl --location --request POST "http://${GRAPHQL_IP}/graphql/alpha" \
-             --header 'Content-Type: application/json' \
-             --data-raw '{"query":"{  account(input: {entityId: {shard: 0, realm: 0, num: 98}}) {alias autoRenewPeriod createdTimestamp declineReward createdTimestamp}}"}'
-
-      {{ end -}}
       {{ if .Values.grpc.enabled -}}
       To access the GRPC API:
 

